### DISCUSSION TITLE
Version 0.1

### DISCUSSION THREAD

**Discussion by LwRed** [SebLague/Geographical-Adventures#11] [General]
Hello, and thanks for this awesome work.  Is it possible to share the 0.1 version with blured clouds ?

**Comment by SebLague** [comment]
Hi, you can find the early version over here (requires a bit of setup): https://www.patreon.com/posts/project-files-58170657

**Reply by LwRed** [reply]
Thanks a lot !

**Comment by johnnyraiden** [comment]
You could make a score based on the distance that you got from the city you should deliver the package to, so then there's a more clear objective, kinda like a geoguesser
